### PR TITLE
avalonia: Add the new Avalonia extension as recommendation

### LINKED ISSUE #102: avalonia: test and add the new Avalonia VS Code extension as recommended extension
Check the https://marketplace.visualstudio.com/items?itemName=AvaloniaTeam.vscode-avalonia

### PR DESCRIPTION
**Pull request by microhobby**
fixes: #102

### CODE REVIEW

**Comment by andreriesco** [comment]
LGTM

File: dotnetAvalonia/.conf/deps.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
         "openssh-client",
         "sshpass",
         "dotnet-sdk-6.0",
+        "dotnet-sdk-7.0",
```
Comment (andreriesco) [inline dotnetAvalonia/.conf/deps.json:6]: Should it be dependent on both "dotnet-sdk-7.0" and "dotnet-sdk-6.0"?

File: dotnetAvalonia/.conf/update.json [hunk 2]
```diff
@@ -2,5 +2,41 @@
...
+        "target": "App.axaml"
+    },
+    {
+        "source": "Views/MainWindow.axaml.cs",
+        "target": "Views/MainWindow.axaml.cs"
+    },
+    {
+        "source": "Views/MainWindow.axaml",
+        "target": "Views/MainWindow.axaml"
     }
```
Comment (andreriesco) [inline dotnetAvalonia/.conf/update.json:41]: Should we also have `app.manifest` and `global.json` here?

Comment (microhobby) [reply dotnetAvalonia/.conf/update.json:41]: they are there: line 15 and line 19

File: dotnetAvalonia/.conf/deps.json (hunk 1 above)
Comment (microhobby) [reply dotnetAvalonia/.conf/deps.json:6]: yeah, because the application will be compiled using the dotnet 6, but the extension use dotnet 7

**Pull request merged**